src/base/utils/misc.cpp

```diff
@@ -445,29 +446,37 @@ QString Utils::Misc::boostVersionString()
 {
     // static initialization for usage in signal handler
     static const QString ver = QString("%1.%2.%3")
-                               .arg(BOOST_VERSION / 100000)
-                               .arg((BOOST_VERSION / 100) % 1000)
-                               .arg(BOOST_VERSION % 100);
+        .arg(QString::number(BOOST_VERSION / 100000)
+            , QString::number((BOOST_VERSION / 100) % 1000)
+            , QString::number(BOOST_VERSION % 100));
     return ver;
 }
 
 QString Utils::Misc::libtorrentVersionString()
 {
     // static initialization for usage in signal handler
-    static const QString ver = LIBTORRENT_VERSION;
-    return ver;
+#if (LIBTORRENT_VERSION_NUM < 10200)
+    static const QString version {QLatin1String {libtorrent::version()}};
```

Comment:
It's better to use `QString::fromLatin1()` when you explicitly want to create QString.

Reply:
You are right, changed.